example/src/App.tsx

```diff
@@ -45,7 +51,7 @@ export default function App() {
               title="Get PID (SPID DEMO)"
```

Comment:
Let's remove this (DEMO) label, what do you think?

Reply:
Being able to use both a production and testing environment and the purpose being a demo in itself, I would leave the label.

Reply:
I added this label and it was referring to the demo environment of the IDP. However we can leave it as it is.

Reply:
Ok. If you think this change is mandatory for delivery feel free to modify the sample app as you see fit.